interface/orders/receive_hl7_results.inc.php

```diff
@@ -823,13 +823,16 @@ function receive_hl7_results(&$hl7, &$matchreq, $lab_id = 0, $direction = 'B', $
                     $in_orderid = intval($porow['procedure_order_id']);
                 }
 
-                if (!$in_orderid) {
-                    // Create order.
+                if (!$in_orderid || $orc) {
+                    // Create order.  Also create a "new" order for each OBR using $orc flag
+                    // since the external order id (filler #) maybe reused due to the inclusion of an ORC segment.
+                    // https://corepointhealth.com/resource-center/hl7-resources/hl7-obr-segment
```

Comment:
Sorry I still don't understand. Why do you want to create a new order if the one that requested this test already exists?

Reply:
hi @sunsetsystems , it wasn't displaying the result in a satisfactory way for lab approval so I came up with this work around for this results only interface.  Without creating a new order the current code combines the result's notes.  There's a sample hl7 file above in case you're interested.  If you'd like I can try to handle this some other way.